app/src/componentLibrary/plainTable/README.md

```diff
@@ -0,0 +1,10 @@
+## **PlainTable**
+
+Has 930px width, min-height 80px
```

Comment:
Please update accordingly.
The height `80px` will be present in case of 1 row?

Reply:
Oh, it is the height of one row without content.
I will fix the width and height values.